### PR TITLE
Do not notify new msgs if chat is muted

### PR DESCRIPTION
**Pull request by paul-nameless**
Mark muted chats
Fix bug with msgs longer than status's width
Add vertical line separtor between chats and msgs
Update chat view

### CODE REVIEW

File: tg/views/__init__.py [hunk 1]
```diff
@@ -82,7 +82,7 @@ def draw(self, msg: Optional[str] = None) -> None:
         self.win.clear()
         if not msg:
             return
-        self.win.addstr(0, 0, msg[: self.w])
+        self.win.addstr(0, 0, msg[: self.w - 1])
```
Comment (paul-nameless) [inline tg/views/__init__.py:85]: Fixed status line failing when msg longer then width

File: tg/views/__init__.py [hunk 2]
```diff
@@ -197,11 +203,11 @@ def draw(self, current: int, msgs: Any) -> None:
             dt, user_id, msg = self._parse_msg(msg)
             user_id = self._get_user_by_id(user_id)
             msg = msg.replace("\n", " ")
-            # remove utf-8 characters that take > 1 bytes to print
+            # count wide character utf-8 symbols that take > 1 bytes to print
             # it causes invalid offset
-            msg = emoji_pattern.sub(r"?", msg)
+            wide_char_len = sum(map(len, emoji_pattern.findall(msg)))
```
Comment (paul-nameless) [inline tg/views/__init__.py:208]: Fix for countring utf-8 wide characters

File: tg/controllers/__init__.py [hunk 3]
```diff
@@ -260,6 +260,21 @@ def update_new_msg(self, update):
         if msg.file_id and msg.size <= config.max_download_size:
             self.download(msg.file_id, chat_id, msg["id"])
 
+        # do not notify, if muted
+        # TODO: optimize
+        chat = None
+        for i, chat in enumerate(self.model.chats.chats):
```
Comment (lightme16) [inline tg/controllers/__init__.py:266]: you don't need `enumerate`

File: tg/controllers/__init__.py [hunk 4]
```diff
@@ -260,6 +260,21 @@ def update_new_msg(self, update):
         if msg.file_id and msg.size <= config.max_download_size:
             self.download(msg.file_id, chat_id, msg["id"])
 
+        # do not notify, if muted
+        # TODO: optimize
+        chat = None
+        for chat in self.model.chats.chats:
+            if chat_id == chat["id"]:
+                chat = chat
```
Comment (lightme16) [inline tg/controllers/__init__.py:268]: just saw it `chat = chat`, that's genius :slightly_smiling_face:

**Pull request merged**